README.md

```diff
@@ -1,4 +1,10 @@
-# wikit
+# WiKit
+[![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lerna.js.org/)
+[![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
+  
+![github](https://github.com/wmde/wikit/workflows/Build%20and%20Deploy%20documentation/badge.svg)
+![github](https://github.com/wmde/wikit/workflows/Test%20and%20deploy/badge.svg)
```

Comment:
If those are links I guess it makes sense they link somewhere a user could want to go.
What about chromatic & the github page respectively?
Alternatively, we could also show a badge with no link.